app/services/work_reminder_generator.rb

```diff
@@ -5,29 +5,37 @@
 class WorkReminderGenerator
   extend T::Sig
 
-  # Sends the day's reminders about open first drafts, using default values for the notification
+  # Sends the day's reminders about open drafts, using default values for the notification
   # interval unless the caller overrides the defaults with custom values. Intended to be run daily
   # by a cron job, but the optional manual override for notification interval is useful in case a
   # human has to manually call the method to send dropped notifications, e.g. in the event that
   # something goes awry with the cron job on a particular day. For example, if the cron job got
   # wedged 2 days ago, you could call this method with the the default values plus 2.
   # @note Intervals are specified in days.
-  sig { params(first_interval: T.nilable(Integer), subsequent_interval: T.nilable(Integer)).void }
-  def self.send_first_draft_reminders(first_interval: nil, subsequent_interval: nil)
-    first_interval ||= Settings.notifications.first_draft_reminder.first_interval
-    subsequent_interval ||= Settings.notifications.first_draft_reminder.subsequent_interval
+  sig { void }
+  def self.send_draft_reminders
+    first_interval = Settings.notifications.first_draft_reminder.first_interval
+    subsequent_interval = Settings.notifications.first_draft_reminder.subsequent_interval
 
-    eligible_works(first_interval, subsequent_interval).find_each do |work|
-      WorksMailer.with(work: work)
-                 .first_draft_reminder_email.deliver_later
+    eligible_works(first_interval, subsequent_interval).each do |state, scope|
+      scope.find_each do |work|
+        mailer = WorksMailer.with(work: work)
+        case state
+        when :first_draft
+          mailer.first_draft_reminder_email.deliver_later
+        when :version_draft
+          mailer.new_version_reminder_email.deliver_later
+        end
+      end
     end
   end
 
-  sig { params(first_interval: Integer, subsequent_interval: Integer).returns(ActiveRecord::Relation) }
+  sig { params(first_interval: Integer, subsequent_interval: Integer).returns(T::Hash[Symbol, ActiveRecord::Relation]) }
   private_class_method def self.eligible_works(first_interval, subsequent_interval)
-    Work
-      .with_state(:first_draft)
-      .where('(((CURRENT_DATE - CAST(created_at AS DATE)) - :first_interval) % :subsequent_interval) = 0',
-             first_interval: first_interval, subsequent_interval: subsequent_interval)
+    %i[first_draft version_draft].index_with do |state|
+      Work.with_state(state)
+          .where('(((CURRENT_DATE - CAST(created_at AS DATE)) - :first_interval) % :subsequent_interval) = 0',
```

Comment:
Can you add a comment that gives an example of these values so it's a little clearer what's going on here?

Reply:
@jmartin-sul actually came up with this approach, so maybe he could explain what's going on. I've just added a new state to the mix.

Reply:
@jcoyne works for me if @jmartin-sul would consider a follow-up PR